src/types/config.ts

```diff
@@ -0,0 +1,29 @@
+import { TheGraphApiImplParams } from 'api/thegraph/TheGraphApi'
+import { MultiTcrApiParams } from 'api/tcr/MultiTcrApi'
+import { DexPriceEstimatorParams } from 'api/dexPriceEstimator/DexPriceEstimatorApi'
+
+export type MultiTcrConfig = {
```

Comment:
Why `type` when we usually do `interface` for such objects?

Reply:
no idea, changing!